assets/src/components/map.tsx

```diff
@@ -125,7 +125,13 @@ const StreetViewControl = ({
           type="checkbox"
           role="switch"
           checked={streetViewEnabled}
-          onChange={() => setStreetViewEnabled((enabled) => !enabled)}
+          onChange={() => {
+            if (!streetViewEnabled && window.FS) {
+              window.FS.event("Dedicated street view enabled")
+            }
```

Comment:
I dunno how this would look in the fullstory UI, but given that it accepts structured data as well, maybe we could use metadata attributes to communicate state instead?

Also, I think the `?.` operator would remove the need for checking if it exists first?

```suggestion
            window.FS?.event("Dedicated street view", { enabled: streetViewEnabled})
```

Reply:
I guess when I read this acceptance criteria:

> Custom event that tracks when any dispatcher toggles the dedicated Street View entrypoint

I mentally filled that in to mean tracking when a dispatcher toggles dedicated Street View _on_, which may not be quite correct. I'll ask product + design.

Reply:
I did just change all of the calls to use `window.FS?.event` rather than wrapping them in an `if` statement in c1d2f7fd.

Reply:
...and changes to the event itself are in 72d348af!